pipeline/src/transformers/addressables/exhibition.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  asText,
+  asTitle,
+  isFilledLinkToDocumentWithData,
+  isNotUndefined,
+} from '@weco/content-pipeline/src/helpers/type-guards';
+import { ExhibitionPrismicDocument } from '@weco/content-pipeline/src/types/prismic';
+import { ElasticsearchAddressableExhibition } from '@weco/content-pipeline/src/types/transformed';
+
+export const transformAddressableExhibition = (
+  document: ExhibitionPrismicDocument
+): ElasticsearchAddressableExhibition => {
+  const { data, id, uid: documentUid } = document;
+  const format = isFilledLinkToDocumentWithData(data.format)
+    ? asText(data.format.data.title)
+    : 'Exhibition';
+
+  const dates = {
+    start: data.start,
+    end: data.end,
+  };
+
+  const primaryImage = data.promo?.[0]?.primary;
+  const description = primaryImage?.caption && asText(primaryImage.caption);
+  const queryDescription = [description, format].filter(isNotUndefined);
+  const title = asTitle(data.title);
+  const uid = documentUid || undefined;
+
+  return {
+    id,
+    uid,
+    display: {
+      type: 'Exhibition',
+      id,
+      uid,
+      title,
+      description,
+      format,
+      dates,
+    },
+    query: {
```

Comment:
Format and contributors missing (We were thinking curators might be listed, or artists)

Reply:
It's why we do this! 💪

Reply:
Looks like I'd added `format` as part of the `queryDescription` (since it isn't an allowed key on `AddressableQuery`). Will add contributors though.

Reply:
ah of course! thanks!